biomejs/biome
# docs(website): portuguese translation

## Summary
Partial work of #1065 which translates the following files:
- `index.md`
- `404.md`
- `linter/index.mdx`
- `guides/getting-started.mdx`





## Test Plan



- Check if there is no missing translation in the translated files.
- Check if the translations are correct and if they adhere to the grammatical rules of Portuguese.
